scalyr_agent/scalyr_monitor.py

```diff
@@ -930,35 +932,35 @@ def get(
                 )
 
             return result
-        except BadConfiguration, e:
+        except BadConfiguration as e:
             raise BadMonitorConfiguration(message=e.message, field=e.field)
 
     def __iter__(self):
-        return self.__map.iterkeys()
+        return six.iterkeys(self.__map)
```

Comment:
Add a TODO to see if we can remove the calls to `six` here for this and the other `iter` related methods.  Like in the other place, I suspect that we can just return the value directly from `self.__map`.

Reply:
If I understood you correctly and you want to return `self.__map__` from `__iter__`,  `__iter__` method can not return non-iterator objects.

Reply:
Gotcha.